pkg/server/helpers.go

```diff
@@ -405,27 +405,17 @@ func isInCRIMounts(dst string, mounts []*runtime.Mount) bool {
 	return false
 }
 
-//TODO: Replace with `mount.Lookup()`in containerd after #257 is marged
-func getMountInfo(mountInfos []*mount.Info, dir string) *mount.Info {
-	for _, m := range mountInfos {
-		if m.Mountpoint == dir {
-			return m
-		}
-	}
-	return nil
-}
-
-func getSourceMount(source string, mountInfos []*mount.Info) (string, string, error) {
-	mountinfo := getMountInfo(mountInfos, source)
-	if mountinfo != nil {
+func getSourceMount(source string, lookupMount func(string) (mount.Info, error)) (string, string, error) {
+	mountinfo, err := lookupMount(source)
+	if err == nil {
 		return source, mountinfo.Optional, nil
 	}
 
 	path := source
 	for {
```

Comment:
@Random-Liu Do you mean this loop?

Reply:
Yeah

Reply:
Seems not necessary, I will remove this loop, Thanks :-)